presto-teradata-functions/src/test/java/com/facebook/presto/teradata/functions/TestTeradataDateFunctions.java

```diff
@@ -93,16 +93,23 @@ public void testYY()
         assertVarchar("to_char(TIMESTAMP '1988-04-08','yy')", "88");
         assertTimestamp("to_timestamp('88/04/08','yy/mm/dd')", 2088, 4, 8, 0, 0, 0);
         assertDate("to_date('88/04/08','yy/mm/dd')", 2088, 4, 8);
+
+        assertTimestamp("to_timestamp('00/04/08','yy/mm/dd')", 2000, 4, 8, 0, 0, 0);
+        assertTimestamp("to_timestamp('20/04/08','yy/mm/dd')", 2020, 4, 8, 0, 0, 0);
+        assertTimestamp("to_timestamp('40/04/08','yy/mm/dd')", 2040, 4, 8, 0, 0, 0);
+        assertTimestamp("to_timestamp('60/04/08','yy/mm/dd')", 2060, 4, 8, 0, 0, 0);
+        assertTimestamp("to_timestamp('80/04/08','yy/mm/dd')", 2080, 4, 8, 0, 0, 0);
+        assertTimestamp("to_timestamp('99/04/08','yy/mm/dd')", 2099, 4, 8, 0, 0, 0);
```

Comment:
aren't these all testing the same thing?

Reply:
Thank you, good point. Removed unnecessary asserts in 2b89c16